isIsogram.js

```diff
@@ -0,0 +1,13 @@
+export function isIsogram(str) {
+    for (let i = 0; i < str.length; i++) {
+
```

Comment:
[nitpick] This is more of a style guide nitpick, but normally empty lines should separate blocks of the same indent. It also is inconsistent with line 7 not having a new line. Conclusion: typo:)

Reply:
Have only added empty lines between blocks of the same indent

![image](https://github.com/kirithecat/MoniquesPlaygroundOne/assets/36695580/9567c7de-74dd-42a4-ba51-486e925676b5)